elements/ip/flowrrswitch.hh

```diff
@@ -0,0 +1,159 @@
+#ifndef CLICK_FLOWRRSWITCH_HH
+#define CLICK_FLOWRRSWITCH_HH
+
+#include <click/batchelement.hh>
+#include <click/ipflowid.hh>
+#include <click/hashtable.hh>
+
+CLICK_DECLS
+
+/*
+ * =c
+ * FlowRRSwitch()
+ *
+ * =s classification
+ * splits flows across its ports in a round-robin fashion
+ *
+ * =d
+ * Can have any number of outputs.
+ * Chooses the output on which to emit each flow based on
+ * a round-robin scheme across the number of output ports.
+ * The element use a hashtable to remember each 5-tuple
+ * passing by and is therefore *not* thread-safe.
+ *
+ * =e
+ * This element expects IP packets and chooses the output
+ * by applying round-robin
+ *
```

Comment:
Requires documentation about input arguments and handlers

Reply:
Actually I think the load aware mode should be a different element. FlowLoadSwitch? FlowMinSwitch?

Reply:
FlowLoadAwareSwitch?

Reply:
LoadAware is not super clear in what sense it is aware. MinLoad? MinPackets? I liked the Min because it shows clearly which output will be selected.

Reply:
FlowMinLoadSwitch then sounds good